src/main/java/com/mineboundteam/minebound/magic/UtilitySpells/TelekineticUtilitySpell.java

```diff
@@ -84,28 +75,32 @@ public void appendHoverText(ItemStack pStack, @Nullable Level pLevel, List<Compo
                                        .append(new TextComponent(manaCost + " Mana").withStyle(ChatFormatting.BLUE).withStyle(ChatFormatting.UNDERLINE))
                                        .append(" per second of flight").withStyle(ChatFormatting.GRAY));
         pTooltipComponents.add(new TextComponent("Additional copies reduce Mana cost by ").withStyle(ChatFormatting.GRAY)
-                                       .append(new TextComponent((int) (manaCostReductions.get(this.level) * 100) + "%").withStyle(ChatFormatting.BLUE).withStyle(ChatFormatting.UNDERLINE))
+                                       .append(new TextComponent((int) (manaCostReduction * 100) + "%").withStyle(ChatFormatting.BLUE).withStyle(ChatFormatting.UNDERLINE))
                                        .append(" each"));
     }
 
     public static class TelekineticUtilitySpellConfig implements IConfig {
 
         public IntValue MANA_COST;
+        public DoubleValue MANA_COST_REDUCTION;
         public final ArmorTier LEVEL;
 
         private final int manaCost;
+        private final double manaCostReduction;
 
-        public TelekineticUtilitySpellConfig(int manaCost, ArmorTier level) {
+        public TelekineticUtilitySpellConfig(int manaCost, double manaCostReduction, ArmorTier level) {
             this.manaCost = manaCost;
             this.LEVEL = level;
+            this.manaCostReduction = manaCostReduction;
         }
 
         @Override
         public void build(Builder builder) {
-            builder.push("Telekinetic Utility");
-            builder.push("Tier " + LEVEL.getValue());
+            builder.push("Utility");
```

Comment:
You can keep telekinetic utility here, maybe put the level on the same line so that each level has its own separate entry.

Reply:
This is how it currently looks. I think having just `Utility` reads better. Otherwise, it will be `Telekinetic."Telekinetic Utility"`
![image](https://user-images.githubusercontent.com/28123262/218379549-bb2e381b-d2b2-4765-a466-388d18366a91.png)

Reply:
Oh I see, alright then